pretyping/reductionops.ml

```diff
@@ -439,10 +418,15 @@ struct
     let n = nargs kargs in
     (List.skipn (n+1) kargs, strip_n_app n stk)
 
+  let expand_case env sigma ((ci, u, pms, t, iv, c) : case_stk) =
+    let dummy = mkProp in
+    let (_, t, _, _, c) = EConstr.expand_case env sigma (ci, u, pms, t, iv, dummy, c) in
+    (t, c)
```

Comment:
shouldn't `c` be called `br` instead?

Reply:
If you want, although α-conversion is quite a thing.

Reply:
Done.